modules/updating-hosted-cluster.adoc

```diff
@@ -43,15 +43,16 @@ $ oc get -n <hosted_cluster_namespace> hostedcluster <hosted_cluster_name> -o ya
 [source,yaml]
 ----
 status:
-  conditions:
-  - lastTransitionTime: "2024-05-20T15:01:01Z"
-	       message: Payload loaded version="4.15.14" image="quay.io/openshift-release-dev/ocp-release:4.15.14-x86_64"
-	       status: "True"
-	       type: ClusterVersionReleaseAccepted
+ conditions:
+ - lastTransitionTime: "2024-05-20T15:01:01Z"
+        message: Payload loaded version="4.x.y" image="quay.io/openshift-release-dev/ocp-release:4.x.y-x86_64" <1>
```

Comment:
```suggestion
        message: Payload loaded version="4.y.z" image="quay.io/openshift-release-dev/ocp-release:4.y.z-x86_64" <1>
```
I think that we write micro releases as x.y.z, according to the [Red Hat supplementary guide](https://redhat-documentation.github.io/supplementary-style-guide/). Since the four takes the place of the x, I think the following letters would be y and z.